pkg/log/logger_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright © 2021 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package log_test
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+	"github.com/suse/elemental/v3/pkg/log"
+)
+
+func TestLogSuite(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Log test suite")
+}
+
+var _ = Describe("logger", Label("log"), func() {
+	It("TestNewLogger returns a logger interface", func() {
```

Comment:
The descriptions now have to be adjusted to no longer reflect the old function names.